.editorconfig

```diff
@@ -0,0 +1,13 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+continuation_indent_size = 2
+indent_size = 2
+
```

Comment:
e.g.: Editors that understand `.editorconfig` can *automatically* format code

(^config matches existing rules used by ESLint)